EIPS/eip-234.md

```diff
@@ -4,8 +4,7 @@ title: Add `blockHash` to JSON-RPC filter options.
 author: Micah Zoltu (@MicahZoltu)
 type: Standards Track
 category: Interface
-status: Last Call
-review-period-end: 2020-08-08
+status: Review
```

Comment:
```suggestion
discussions-to: https://github.com/ethereum/EIPs/issues/234
status: Last Call
review-period-end: 2021-06-01
```

Reply:
This EIP is ancient, I'm surprised it wasn't merged given how it has long since been implemented.